app/car.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import json
+import math
+
+from typing import List
+from decimal import Decimal
+
+
+class Car:
+    def __init__(self, dictionary: dict) -> None:
+        self.brand = dictionary["brand"]
```

Comment:
it's preferable to use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError